packages/ui/src/ui/entries/ytm-overrides.scss

```diff
@@ -0,0 +1,113 @@
+// Overrides for styles from @diplodoc/transform/dist/css/yfm.css
```

Comment:
A misprint in the filename: `ytm-overrides.scss` -> `yfm-overrides.scss`. 
Is it works for you?

Reply:
Fixed, sorry.